PWGHF/Tasks/taskD0parametrizedPID.cxx

```diff
@@ -113,13 +118,44 @@ struct TaskD0parametrizedPIDMC {
       //if (ncontributor<=centralitySelectionMin && ncontributor>centralitySelectionMax) {
       //  continue;
       //}
+      Double_t maxFiducialY = 0.8;
+      Double_t minFiducialY = -0.8;
```

Comment:
Why do you not use `float`?

Reply:
Do you have a reason for using `Float_t` (which is a ROOT type) instead of `float` (which is a native C++ type)?